src/tests.builds

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ExcludeProjects Condition="'$(OSEnvironment)'!='Windows_NT'" Include="**\Microsoft.VisualBasic.Tests.csproj" />
+    <ExcludeProjects Condition="'$(OSEnvironment)'=='Windows_NT'" Include="**\System.Security.Cryptography.OpenSsl.Tests.csproj" />
```

Comment:
The UnsupportedPlatforms tag does not prevent the assembly from being built, which causes tests to build product binaries that should not be built for a specific platform (ie OpenSsl). Right now, we need a quick fix to get the OpenSsl failures from showing up. PR #6081 was going to be a full-fledged fix for the whole repo but that ended up being a lot more work and a lot more intricate than I currently have time for, so this is a targeted fix for this specific binary